h/services/user_signup.py

```diff
@@ -43,13 +43,13 @@ def signup(self, require_activation=True, **kwargs):
         :param require_activation: The name to use.
         :type require_activation: bool.
 
-        :param identities: A list of dictionaries representing identities to
-          add to the new user. Each dictionary will be passed as keyword args
-          to :py:class:`h.models.UserIdentity`.
-
         Remaining keyword arguments are used to construct a new
         :py:class:`h.models.User` object.
 
+            * *identities*  A list of dictionaries representing identities to
+              add to the new user. Each dictionary will be passed as keyword args
+              to :py:class:`h.models.UserIdentity`.
```

Comment:
```suggestion
              to `h.models.UserIdentity`.
```

I think we're scooping these out now when we see them?